### PR TITLE
Dry repetitious calculations

### PR DESCRIPTION
**Pull request by seriousbusinessguy**
also use integer arithmetic

### CODE REVIEW

File: Challenge07/src/challenge07/Challenge07.java [hunk 1]
```diff
@@ -12,50 +12,55 @@
...
-        double purchaseCountyTax;       //County tax on purchase
-        double totalTax;                //Total tax on purchase
-        double purchasePrice;           //User-entered purchase price
-        double totalPrice;              //The total price to be displayed
+        final double STATE_TAX = 0.04;    //State tax rate
+        final double COUNTY_TAX = 0.02;   //County tax rate
+        Integer purchaseStateTax;        //State tax on purchase
+        Integer purchaseCountyTax;       //County tax on purchase
+        Integer totalTax;                //Total tax on purchase
+        Integer purchasePrice;           //User-entered purchase price
```
[the remark this answers is not in the document]
Comment (kennedymeadows) [reply Challenge07/src/challenge07/Challenge07.java:30]: nevermind i figured it out

File: Challenge07/src/challenge07/Challenge07.java [hunk 2]
```diff
@@ -12,50 +12,55 @@
...
+        Integer totalPrice;              //The total price to be displayed
         
         System.out.print("How much is the snow mobile? $");  
-                        //To find purchase price
-            purchasePrice = keyboard.nextDouble();           //User input
+        //To find purchase price
+        double rawPurchasePrice = keyboard.nextDouble(); //User input
+
+        purchasePrice = Math.round(rawPurchasePrice * 100);
```
Comment (kennedymeadows) [inline Challenge07/src/challenge07/Challenge07.java:37]: Math.round why do you have to use this method? why not just multiple raw * 100?

File: Challenge07/src/challenge07/Challenge07.java [hunk 3]
```diff
@@ -12,50 +12,55 @@
...
+
+
             
-            purchaseStateTax = purchasePrice * STATE_TAX;    
+            purchaseStateTax = Math.round(purchasePrice * STATE_TAX);
```
Comment (kennedymeadows) [inline Challenge07/src/challenge07/Challenge07.java:41]: same here

File: Challenge07/src/challenge07/Challenge07.java (hunk 2 above)
Comment (seriousbusinessguy) [reply Challenge07/src/challenge07/Challenge07.java:37]: maybe that works? can you assign a double to an integer and get auto-casting?

File: Challenge07/src/challenge07/Challenge07.java [hunk 4]
```diff
@@ -12,50 +12,55 @@
...
-            purchasePrice = keyboard.nextDouble();           //User input
+        //To find purchase price
+        double rawPurchasePrice = keyboard.nextDouble(); //User input
+
+        purchasePrice = (int)(rawPurchasePrice * 100);
+
+
             
-            purchaseStateTax = purchasePrice * STATE_TAX;    
+            purchaseStateTax = (int)(purchasePrice * STATE_TAX);
```
Comment (kennedymeadows) [inline Challenge07/src/challenge07/Challenge07.java:41]: honestly i dont think it needs the (int) here

**Pull request merged**